src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -1,6 +1,9 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using Microsoft.Identity.ServiceEssentials;
```

Comment:
Does this mean MSAL takes a dependency on MISE?

Reply:
No, this is just temporary since that's the namespace where the code is.